gpflow/models/cglb.py

```diff
@@ -181,6 +181,8 @@ def predict_f(
         :param cg_tolerance: float or None: If None, the cached value of
             :math:`v` is used. If float, conjugate gradient is run until :math:`rᵀQ⁻¹r < ϵ`.
         """
+        assert_params_false(self.predict_f, full_output_cov=full_output_cov)
```

Comment:
A pity we have to pass in the method name each time as I'm sure we'll end up with copy and paste errors.

You could probably replace this with a decorator of some sort, though I expect it would confuse mypy.

```python
@assert_param(full_output_cov=False)
def predict_f(...):
```

Reply:
> A pity we have to pass in the method name each time as I'm sure we'll end up with copy and paste errors.
> 
> You could probably replace this with a decorator of some sort, though I expect it would confuse mypy.
> 
> ```python
> @assert_param(full_output_cov=False)
> def predict_f(...):
> ```

Again, I'm more worried about confusing TensorFlow.